src/manage.c

```diff
@@ -177,7 +178,7 @@ u8 best_prn(void)
       if (acq_prn_param[prn].state == ACQ_PRN_TRIED)
         acq_prn_param[prn].state = ACQ_PRN_UNTRIED;
     }
-    printf("acq: restarting PRN search\n");
+    log_info("acq: restarting PRN search\n");
```

Comment:
so, I do find it useful that these messages are prefixed by the subsystem of origin. I think something similar could be done with other `log_info`s here (such as the `log_info("PRN ... unhealthy");` line, for example.